submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/service/SubmissionService.kt

```diff
@@ -32,34 +36,50 @@ class SubmissionService(
     fun submit(rqt: SubmitRequest): ExtSubmission {
         logger.info { "${rqt.accNo} ${rqt.owner} Received sync submit request for submission ${rqt.accNo}" }
         val submission = submissionSubmitter.submit(rqt)
-        eventsPublisherService.submissionSubmitted(submission)
+        eventsPublisherService.submissionSubmitted(submission.accNo, submission.owner)
         return submission
     }
 
     fun submitAsync(rqt: SubmitRequest) {
         logger.info { "${rqt.accNo} ${rqt.owner} Received async submit request for submission ${rqt.accNo}" }
+        val (accNo, version) = submissionSubmitter.createRequest(rqt)
+        eventsPublisherService.requestCreated(accNo, version)
+    }
+
+    @RabbitHandler
+    fun loadRequest(rqt: RequestCreated) {
+        processSafely(rqt) {
+            logger.info { "$accNo, received Created message for submission $accNo, version: $accNo" }
```

Comment:
We're supposed to include the owner in the logs for kibana indexing purposes: 

$accNo $owner <message>

Reply:
request message does not contain owner.  I do not think is worth it to include it only for logging purposes.